detect/buildpacks.go

```diff
@@ -204,4 +204,40 @@ var _ = DetectDescribe("Buildpacks", func() {
 			})
 		}
 	})
+
+	Describe("nginx", func() {
+		for _, stack := range Config.GetStacks() {
```

Comment:
`stack` should be assigned to a local copy inside of the for loop, or else "the It closure will capture the mutating loop variable and all the specs will run against the last element in the loop." (re: [Dynamically Generating Specs](https://onsi.github.io/ginkgo/#dynamically-generating-specs))

Reply:
I see that most of our for loops on this page don't do this, and I think that they should all be changed while we're here.

Reply:
Ah ok, I was scratching my head what `stack := stack` should be doing... will fix.